core/src/commonMain/kotlin/com/ustadmobile/core/view/ContentEntryEdit2View.kt

```diff
@@ -22,7 +22,7 @@ interface ContentEntryEdit2View: UstadEditView<ContentEntryWithLanguage>{
 
     var entryMetaData: ImportedContentEntryMetaData?
 
-    suspend fun saveContainerOnExit(entryUid: Long,selectedBaseDir: String, db: UmAppDatabase, repo: UmAppDatabase): Container?
+    fun unregisterFileFromTemp()
```

Comment:
This isn't really a view property or function (that was also the case for saveContainerOnExit). I think it would be better to keep that in Android. I would suggest overriding the onClickOptionsMenu so that you capture the click of the save menu item. Then you can unregister the temp file on the Android side.